src/Microsoft.Health.Dicom.Core/Features/Export/IdentifierExportSource.cs

```diff
@@ -45,12 +45,12 @@ public async IAsyncEnumerator<ReadResult> GetAsyncEnumerator(CancellationToken c
         foreach (DicomIdentifier identifier in _identifiers)
```

Comment:
Did I not use `IEnumerable<ReadResult> results = Enumerable.Empty<ReadResult>();`? Delete it